edi_835_parser/transaction_set/transaction_set.py

```diff
@@ -864,6 +875,16 @@ def count_services(self) -> int:
 				count += len(claim.services)
 
 		return count
+	
+	def sum_interests(self):
```

Comment:
is this called anywhere within the package, or is the intent for this to be called by the end user?

similar to my comment above, if `interest_amount` was a property on the claim loop, that could be added as an attribute in the claim in [this part of the code](https://github.com/CoveHealth/edi-835-parser/pull/3/files?diff=split&w=1#diff-12360646ee84ee0e71ebc953ab52af6cd14965fc0a15009b86abf7b84a62b45fR303) vs having claim-specific logic handled at the transaction set level

Reply:
alternatively, if you think both types of amounts are ultimately going to be treated the same, these could be consolidated down into a single `total_amount` property on the claim loop where we total all the amounts, regardless of the qualifier code

Reply:
yea I just created this method for the convenience of unit tests, I dont think we would actually use it in our package or use it in the `process_change_remittance_report` task

Reply:
if it's code we don't want people to use, can we put the logic into the tests instead of in the main interface for this class?

Reply:
sounds good, I moved all the existing test functions to the test file as well